course_contents/2_intro_to_python/notes/01. Conditions.md

```diff
@@ -0,0 +1,101 @@
+### The `if` statement
+- the `if` statement is used to check if a conditional is `True`
+- syntax for writing the `if` statement:
+    1. use the `if` keyword at the beginning of the statement
+    2. follow the keyword with a correctly written conditional
+    3. finish the expression with colon 
+    4. block of code that will be executed if the statement passes - should be indented (usually by 4 spaces)
+
+An example of proper writing of the `if` statement:
+
+```py
+friend = "Rolf"
+name = input("Enter your name: ")
+if name == friend:
+    print(f"{name} is a friend")
+```
+
+Two examples of incorrect writing of the `if` statement:
+
+```py
+if name = friend:
```

Comment:
I would point out the mistakes for people that don't see them

Reply:
I was guided by our motto that we remind people of stuff rather than explain it to them, and the refresher seems too long right now so adding more explanation could make it even worse.
I can add it, and I agree that it will make it easier to understand for our students, but I think that @jslvtr should decide what we want to do in terms of size vs explanation side of things

Reply:
I think pointing out in the design, with an arrow or a small callout (as noted now) will be sufficient! Nice one :+1: